src/main/java/org/fundacion/jala/converter/service/ExtractText.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package org.fundacion.jala.converter.service;
+import net.sourceforge.tess4j.Tesseract;
+import net.sourceforge.tess4j.TesseractException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+
+public class ExtractText {
+
+    private String language;
+    private String pathFile;
+    private String nameOutputFile;
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    public ExtractText(final String language, final String pathFile) {
+        this.language = language;
+        this.pathFile = pathFile;
+    }
+
+    public ExtractText(final String language, final String pathFile, final String nameOutputFile) {
+        this.language = language;
+        this.pathFile = pathFile;
+        this.nameOutputFile = nameOutputFile;
+    }
+
+    /**
+     *
```

Comment:
This line would be unnecesary

Reply:
I put that space because checkstyle required it, I will try to remove it